lib/types/module/ModuleBuilder.js

```diff
@@ -0,0 +1,27 @@
+const AbstractBuilder = require("../AbstractBuilder");
+const tasks = { // can't require index.js due to circular dependency
+	uglify: require("../../tasks/uglify")
+};
+
+class ModuleBuilder extends AbstractBuilder {
+	constructor({resourceCollections, project, parentLogger}) {
+		super({project, parentLogger});
+
+		// All available library tasks in execution order
+		this.availableTasks = [
+			"uglify"
+		];
+
+		this.addTask("uglify", () => {
```

Comment:
Do we really want to always uglify?
I would expect that modules from the npm registry are already optimized.

Reply:
Done